tests/interfaces/test_torch.py

```diff
@@ -632,30 +632,18 @@ def expand(self):
                 ]
                 return tape
 
-        tape = qml.tape.QuantumTape()
-
         dev = qml.device("default.qubit", wires=1)
         a = np.array(0.1)
         p_val = [0.1, 0.2, 0.3]
         p = torch.tensor(p_val, requires_grad=True, device=torch_device)
 
-        with tape:
+        with qml.tape.QuantumTape() as tape:
             qml.RX(a, wires=0)
             U3(p[0], p[1], p[2], wires=0)
             qml.expval(qml.PauliX(0))
 
-        tape = tape.expand()
         res = execute([tape], dev, **execute_kwargs)[0]
 
-        assert tape.trainable_params == [1, 2, 3, 4]

```

Comment:
`trainable_params` aren't set on the tape we pass in, since its expanded under the hood.